Unable to filter by float/fixed client

sbuser-git opened the pull request:
Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

<!--
Thank you for your interest in and contributing to IBM Spectrum LSF RTM! There are a few simple things to check before submitting your pull request that can help with the review process. You should delete these items from your submission, but they are here to help bring them to your
attention.
-->
<!-- If this pull request resolves any bugs in the issues, provide a link: -->
Resolved #XXXX.
<!--
Have you followed the [contributor guidelines](https://github.com/IBM/ibm-spectrum-lsf-rtm-server/blob/main/CONTRIBUTING.md)?
Thank you for your contribution to IBM Spectrum LSF RTM!
-->

sbuser-git committed:
Unable to filter by float/fixed client

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

ddb4github commented:
Extra changed required:
 first for performance, second for terminology consistence.
```diff
	if (get_request_var('type') > 0) {
		if (get_request_var('type') == 1) {
			$sql_where .= ($sql_where != '' ? ' AND ':'WHERE ') . 'licFeaturesNeeded = 16';
-		} else {
-			$sql_where .= ($sql_where != '' ? ' AND ':'WHERE ') . 'licFeaturesNeeded != 16';
+		} else if (get_request_var('type') == 2) {
+			$sql_where .= ($sql_where != '' ? ' AND ':'WHERE ') . 'licFeaturesNeeded == 512';
		}
	}

@@ -101,8 +101,8 @@ function clientsFilter() {
							<?php
							$types = array(
								-1 => __esc('All Clients', 'grid'),
-								1  => __esc('Fixed Client', 'grid'),
-								2  => __esc('Float Client', 'grid')
+								1  => __esc('Fixed Clients', 'grid'),
+								2  => __esc('Float Clients', 'grid')
							);

							if (cacti_sizeof($types)) {
```

sbuser-git committed:
Update grid_clients.php

Per Jing, updated w/extra changes: first for performance, second for terminology consistence.

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

sbuser-git reviewed (commented):
Addressed "Extra changed required: first for performance, second for terminology consistence."
